www/js/lib/fileSystem.js

```diff
@@ -0,0 +1,238 @@
+// refer to this article for easy explanation of File System API https://developer.chrome.com/articles/file-system-access/
```

Comment:
Add the same header, but include your GitHub name, as you find at the head of other Kiwix module files. You can use the header from `app.js`, but begin it with `/**` instead of `/*!` (the latter tells rollup.js to include the header in the built version, and we only want the header to appear once in the built version). It should start like this:

```
/**
 * fileSystem.js : BRIEF FILE DESCRIPTION GOES HERE
 * This file handles MORE DESCRIPTION, INCLUDE REFERENCE TO developer.chrome.com FROM ABOVE
 *
 * Copyright 2013-2023 Rishabhg71, Jaifroid and contributors
 * Licence GPL v3:
 *
 * This file is part of Kiwix.
 *
```
(etc.)